### DISCUSSION TITLE
Maximum and average values of the elemental values calculated by "ParsedMaterial"

### DISCUSSION THREAD

**Discussion by wenpeng231024** [idaholab/moose#18485] [Q&A General] [answered]
Dear MOOSE group,

I can use "ParsedMaterial" to obtain the elemental values. How can I obtain the maximum or average value of these elemental values?

Thanks,

Peng

**Comment by GiudGiud** [comment, marked as answer]
Hello

I assume you mean on a subdomain-basis for the maximum and average. 
Please have a look at
https://mooseframework.inl.gov/source/postprocessors/ElementExtremeMaterialProperty.html
https://mooseframework.inl.gov/source/postprocessors/ElementAverageMaterialProperty.html

Guillaume

**Reply by wenpeng231024** [reply]
Yes. Got it. Many thanks.

Peng
